packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1064,9 +1064,29 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
     });
     // If a main meeting call exists then process that call and resume
     if (mainMeetingCall) {
+      const breakoutRoomCall = this.call;
       this.processNewCall(mainMeetingCall);
       await this.resumeCall();
+      if (breakoutRoomCall?.state === 'Connected') {
+        breakoutRoomCall.hangUp();
+      }
+    }
+  }
+
+  /* @conditional-compile-remove(breakout-rooms) */
+  public async hangUpOriginCall(): Promise<void> {
```

Comment:
I guess there is not binding layer change? if so I am wondering if we could just override the hangup function props from composite layer to avoid api changes of component

Reply:
Good obeservation. I will try to override the hangup functions from the composite layer to the handUpOriginCall api change